plugin-server/src/healthcheck.ts

```diff
@@ -21,6 +22,34 @@ export async function healthcheck(): Promise<boolean> {
     }
 }
 
+const storageHealthcheck = async (): Promise<boolean> => {
+    if (!defaultConfig.OBJECT_STORAGE_ENABLED) {
+        return true
+    }
+
+    const storage = connectObjectStorage(defaultConfig)
+    try {
+        const storageHealthy = await storage.healthCheck()
+        if (storageHealthy) {
+            status.info('💚', `object storage is healthy`)
+            return true
+        } else {
+            status.error('💔', 'object storage is not healthy')
+            return false
+        }
+    } catch (error) {
+        status.error('💥', 'Object Storage healthcheck: an unexpected error occurred:', error)
+        return false
+    }
+}
+
+export async function healthcheck(): Promise<boolean> {
```

Comment:
We're not currently using this even - but happy to update it for when we do use it.

Also note that the idea of this healthcheck is that we might take the task down if it doesn't pass - is that what we wish to do with object storage now?